NOTICE

```diff
@@ -91,4 +91,10 @@ This product contains modified versions of the Dockerfile and related configurat
   * HOMEPAGE:
     * https://github.com/sequenceiq/hadoop-docker/
   * COMMIT TAG:
-    * update this when this patch is committed
\ No newline at end of file
+    * update this when this patch is committed
+
+This product contains code adapted from Apache Hive
```

Comment:
there is already one entry for Hive above, to make this more relevant maybe need to add what is copied and commit or release version.

Reply:
Heh, oops I'm dumb and didn't actually read the `NOTICE` file, just knew i should add something since I copied it from elsewhere, will remove this extra :+1: